x/claim/keeper/hooks.go

```diff
@@ -9,27 +9,19 @@ import (
 )
 
 func (k Keeper) AfterAddLiquidity(ctx sdk.Context, sender sdk.AccAddress) {
-	if k.CheckAndSetUserAction(ctx, sender, types.ActionAddLiquidity) {
-		k.ClaimCoins(ctx, sender.String())
-	}
+	k.ClaimCoinsForAction(ctx, sender, types.ActionAddLiquidity)
```

Comment:
Rather than ignoring the error for this part, I think panicking if an error exist may be better. I don't see specific reason this part causing errors, but it seems better to make the transaction fail if there happens to be an error.

Reply:
Done